### PR TITLE
story/BODS-777 + story/BODS-774 + story/BODS-775: Improve Interactive API

### PR DESCRIPTION
**Pull request by ndemenezes**
Fixes:

- Update text descriptions for all query parameters for all api types
- Clean up bounding box options and provide realistic example
- Remove servers DDL from all api types

### CODE REVIEW

File: transit_odp/static/openapi/avl3.yml [hunk 1]
```diff
@@ -36,16 +36,17 @@ paths:
...
+            boundingBox you set using co-ordinates.
+            [minLongitude, minLatitude, maxLongitude, maxLatitude].
+            The default values shown are for the Liverpool area.
           schema:
             type: array
             items:
               type: number
             minItems: 4
             maxItems: 4
-            example: [51.401, 51.509, 0.01, 0.201]
```
Comment (surabhika) [inline transit_odp/static/openapi/avl3.yml:48]: This bounding box change also needs to be done for '/gtfsrtdatafeed'

File: transit_odp/static/openapi/fares3.yml [hunk 2]
```diff
@@ -71,29 +71,28 @@ paths:
             maxItems: 4
             example: [51.401, 51.509, 0.01, 0.201]
           description: >
-            Limit results to fares data sets that contain information for the area within the rectangular boundingBox you set using co-ordinates</br></br>
+            Limit results to fares data sets that contain information for the area within the rectangular boundingBox you set using co-ordinates.</br></br>
```
Comment (surabhika) [inline transit_odp/static/openapi/fares3.yml:74]: Bounding box change needs to be made here too.

**Review by surabhika — CHANGES_REQUESTED** [review]

File: transit_odp/static/openapi/avl3.yml (hunk 1 above)
Comment (ndemenezes) [reply transit_odp/static/openapi/avl3.yml:48]: @surabhika Added these in.

File: transit_odp/static/openapi/fares3.yml (hunk 2 above)
Comment (ndemenezes) [reply transit_odp/static/openapi/fares3.yml]: @surabhika Added these in.

**Review by surabhika — APPROVED** [review]

**Pull request merged**